templates/careers/_search.html

```diff
@@ -141,17 +154,21 @@
 });
 </script>
 {% endif %}
-<div class="js-search-jobs-form u-hide">
-  <form class="p-search-box">
-    <label class="u-off-screen" for="search">Search</label>
-    <input type="search" id="search" class="p-search-box__input js-careers__search-input" name="search" placeholder="Search" required="" autocomplete="off">
-    <button type="reset" class="p-search-box__reset"><i class="p-icon--close">Close</i></button>
-    <button type="submit" class="p-search-box__button"><i class="p-icon--search">Search</i></button>
+<div class="js-search-jobs-form u-hide u-align--bottom" style="display: flex;">
+  <form action="/careers/all" class="p-form p-form--inline p-form--search">
+    <div class="p-form__group" style="flex-direction: column;">
+      <label class="u-off-screen" for="search">Search</label>
+      <div class="p-form__control u-clearfix u-no-padding--right">
+        <input class=" js-careers__search-input" id="search" type="search" name="search" value="" placeholder="Search by keyword" />
```

Comment:
autocomplete needs to be off i think as it looks like this! 
<img width="474" alt="image" src="https://user-images.githubusercontent.com/57550290/194035177-fceca6f2-9d67-4829-a778-66a59e24031f.png">

```suggestion
        <input class=" js-careers__search-input" id="search" type="search" name="search" value="" placeholder="Search by keyword" autocomplete="off" />
```